### ISSUE TITLE
Problem with custom liquid potion recipes

### ISSUE THREAD

**Issue by mathwiz617** [jaredlll08/ModTweaker#671]
#### Issue Description:

When trying to remove the Thermal Expansion Imbuer recipe for basic healing potions, my script threw an error, saying there was an invalid character that the parser couldn't handle.

#### What happens:

An error is thrown, citing the " character.

#### What you expected to happen:

The recipe should be removed.

#### Script used (Please Pastebin or gist your script, posting an unpasted or ungist'd script will automatically close this issue):

https://pastebin.com/F2tLaadk

#### Minetweaker.log file (Please Pastebin or gist your file, posting an unpasted or ungist'd file will automatically close this issue):

https://pastebin.com/psTkWnNW

____
#### Affected Versions (Do *not* use "latest"):

- Minecraft: 1.12.2
- Forge: 14.23.4.2747
- Crafttweaker: CraftTweaker2-1.12-4.1.9
- ModTweaker: 4.0.13
- MTLib: 3.0.5

#### Your most recent log file where the issue was present: 

https://pastebin.com/psTkWnNW

**Comment by mathwiz617** [comment]
This happens with all the "<liquid:potion>.withTag({Potion: "minecraft:[potionName]})" declarations that I try, not just liquid awkward potion.

**Comment by mathwiz617** [comment]
Hmm... I tried typing this in by hand instead of copy/pasting, and got an entirely different error - there's an invalid character at the end of my file. This may be a text editor problem, not a parser problem.

EDIT: Definitely an editor problem - it was replacing regular quote marks with curly quote marks, and that's what the parser was getting hung up on. Also, I changed the encoding, and the invalid character is gone.
